src/Umbraco.Web.UI.Client/src/views/packages/views/repo.controller.js

```diff
@@ -24,6 +24,8 @@
         vm.closeLightbox = closeLightbox;
         vm.search = search;
         vm.installCompleted = false;
+        vm.highlightedPackageCollections = [];
+        vm.labels = [];
```

Comment:
Maybe change this to `vm.labels = {}` instead.

Then we could later do the following:

```
vm.labels.popularPackages  = labels[0];
vm.labels.promotedPackages = labels[1];
```

Reply:
That's nicer certainly - have updated to use that pattern.